contrib/thrift-maven-plugin/pom.xml

```
@@ -94,7 +94,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>32.0.0-jre</version>
+        <version>32.0.1</version>
```

Not sure what's up with the change in spacing, but this needs `-jre` in the version name to resolve.

```suggestion
      <version>32.0.1-jre</version>
```